### ISSUE TITLE
why can't find module?

### ISSUE THREAD

**Issue by AlexKingGo** [soumyakoduri/nfs-test-suite#2]
a lot of module with postfix _pack, _const, _type can't find,  and in the project i can't  find too.

**Comment by soumyakoduri** [comment]
this test-suite was developed with earlier versions of python and I guess thats why you are running into these errors.

**Comment by AlexKingGo** [comment]
thank you .
Because my python is poor and i still be confused that ,for example,in package pynfs/nfs4.1/nfs4client.py  it from xdrdef.nfs4_type import *，from xdrdef.nfs4_const import *. but i can't find the nfs4_const or nfs4_type in package xdrdef.  is this problem revelent with python version?

**Comment by AlexKingGo** [comment]
Sorry i just know the xdr and rpc generator, very thank you for generous answer!  Best wishes to you.

**Comment by soumyakoduri** [comment]
No problem. Its just that its been years since we had used this test-suite. If you are really interested to use it, I can try to fix things in my free time. Otherwise almost all the tests used in this repo are available online and can be run individually.

**Comment by AlexKingGo** [comment]
Yes, i am very interested.Now I am an trainee for just one week, my task is developing a test tool which  simulate that many nfs client access nfs server with nfs v4.1. i never touched rpc programming. I search nfs client in github almost the whole day and your test tool is the most suitable for me ,but it is a little confusing of the directory structure.
